.github/workflows/tarpaulin.yml

```diff
@@ -0,0 +1,66 @@
+name: tarpaulin
```

Comment:
This GHA workflow uses `cargo-tarpaulin` for the coverage determination.  Naming it accordingly seemed intuitive to me.